odl/discr/discretization.py

```diff
@@ -214,6 +212,11 @@ def __eq__(self, other):
                 other.interpolation == self.interpolation)
 
     @property
+    def impl(self):
+        """The underlying implmentation type for the dspace."""
```

Comment:
Id probably simply add ticks to `dspace`, that would create a link to the `dspace` property which explains further?